src/test/python/testcollections.py

```diff
@@ -4,18 +4,18 @@
 import unittest
 import asyncio
 
-import pyrt.windows.foundation.collections as wfc
+import winrt.windows.foundation.collections as wfc
 
 class TestCollections(unittest.TestCase):
 
     def test_stringmap(self):
         m = wfc.StringMap()
-        m.Insert("hello", "world")
+        m.insert("hello", "world")
 
-        self.assertTrue(m.HasKey("hello"))
-        self.assertFalse(m.HasKey("world"))
-        self.assertEqual(m.Size, 1)
-        self.assertEqual(m.Lookup("hello"), "world")
+        self.assertTrue(m.has_key("hello"))
+        self.assertFalse(m.has_key("world"))
+        self.assertEqual(m.size, 1)
+        self.assertEqual(m.lookup("hello"), "world")
```

Comment:
Any specific wrappers for these yet? `__in__`, `__len__`, `__getitem__`?

Reply:
Yeah, IMap/IMapView support mapping protocol. Will add tests